ui/apps/platform/src/Containers/SystemHealth/Components/DeclarativeConfigurationHeathCard.tsx

```diff
@@ -0,0 +1,117 @@
+     * Wait for isFetching only until response to the initial request.
+     * Otherwise count temporarily disappears during each subsequent request.
+     */
+    const hasCount = (pollingCount !== 0 || !isFetching) && !requestErrorMessage;
+    const itemCount = items.length;
+
+    return (
+        <Card isFullHeight isCompact>
+            <CardHeader>
+                <CardHeaderMain>
+                    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                        <FlexItem>
+                            <CardTitle component="h2">Declarative configuration</CardTitle>
+                        </FlexItem>
+                        {hasCount && (
+                            <FlexItem>
+                                <IconText
+                                    icon={
+                                        itemCount === 0 ? (
+                                            <CheckCircleIcon color="var(--pf-global--success-color--100)" />
+                                        ) : (
+                                            <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+                                        )
+                                    }
+                                    text={
+                                        itemCount === 0
+                                            ? 'no errors'
+                                            : `${itemCount} ${pluralize('error', itemCount)}`
+                                    }
+                                />
+                            </FlexItem>
+                        )}
+                    </Flex>
+                </CardHeaderMain>
+            </CardHeader>
+            {(requestErrorMessage || itemCount !== 0) && (
+                <CardBody>
+                    {requestErrorMessage ? (
+                        <Alert isInline variant="warning" title={requestErrorMessage} />
+                    ) : (
+                        <TableComposable variant="compact">
+                            <Thead>
+                                <Tr>
+                                    <Th width={40}>Name</Th>
+                                    <Th width={40}>Error</Th>
+                                    <Th width={20}>Date</Th>
+                                </Tr>
+                            </Thead>
+                            <Tbody>
+                                {items.map(({ id, name, errorMessage, lastTimestamp }) => (
+                                    <Tr key={id}>
+                                        <Td dataLabel="Name" modifier="breakWord">
+                                            {name}
+                                        </Td>
+                                        <Td dataLabel="Error" modifier="breakWord">
+                                            {errorMessage}
+                                        </Td>
+                                        <Td dataLabel="Date">{lastTimestamp}</Td>
```

Comment:
WDYT about leaving milliseconds out of the displayed timestamp? E.g. `2023-04-23T23:18:47Z`